docs/reference-architectures/serverless/web-app.md

```diff
@@ -12,46 +12,34 @@ ms.custom: seodec18, serverless
 
 # Serverless web application on Azure
 
-This reference architecture shows a [serverless](https://azure.microsoft.com/solutions/serverless/) web application. The application serves static content from Azure Blob Storage, and implements an API using Azure Functions. The API reads data from Cosmos DB and returns the results to the web app.
+This reference architecture shows a [serverless](https://azure.microsoft.com/solutions/serverless/) web application. The application serves static content from Azure Storage, and implements an API using Azure Functions. The API reads data from Cosmos DB and returns the results to the web app.
```

Comment:
I've heard feedback from developers that they don't know what "blob" means, and that we use too many acronyms. I think it's ok to leave that out of the conversation until we absolutely have to address it later.